1.illumination_correction/scripts/0.illum_correct.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Run illumination correction on data
+# 
+# Note: We load in the CellProfiler IC pipeline to use for this process.
+
+# ## Import libraries
+
+# In[1]:
+
+
+import pathlib
+import pprint
+import requests
+
+import sys
+
+sys.path.append("../utils")
+import cp_parallel
+
+
+# ## Set paths and variables
+
+# In[2]:
+
+
+# set the run type for the parallelization
+run_name = "illum_correction"
+
+# set main output dir for all plates if it doesn't exist
+output_dir = pathlib.Path("./Corrected_Images")
+output_dir.mkdir(exist_ok=True)
+
+# directory where images are located within folders (held in DropBox specific directory)
+images_dir = pathlib.Path("../../Way Science Lab Dropbox/Jenna  Tomkinson/McKinseyLab_WayLab_CardiacFibroblasts/Compound Screen")
```

Comment:
Agreed, but I am unsure if changing it would break DropBox's ability to sync with `fig` given that there isn't much compatibility. I might test this later to see if there would be any issues but that is beyond the scope for this PR. Thanks for catching and I will add this as an issue.